Feature/upstream interval improvements (#1997)
spatialos/UnrealGDK: CHANGELOG.md

- The `bPackRPCs` property has been removed, and the flag `--OverrideRPCPacking` has been removed.
- Added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and ActorComponents. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
- You can now generate valid schema for classes that start with a leading digit. The generated schema class will be prefixed with `ZZ` internally.

> - Added setting `bWorkerFlushAfterSend` (defaulted true) which schedules the worker flush after the GDK worker queue has been executed.

Reviewer: ```suggestion
- Added `bWorkerFlushAfterSend` (defaulted true) which schedules the worker flush after the GDK worker queue has been executed.
```

> - Added setting `bWorkerFlushAfterRPC` (defaulted false) which wakes the GDK worker queue to allow for lower latencies.

Reviewer: ```suggestion
- Added `bWorkerFlushAfterRPC` (defaulted false) which wakes the GDK worker queue to allow for lower latencies.
```

> - Added setting `bWorkerSendHighPriority` (defaulted false) which sets the GDK worker thread priority higher than normal.

Reviewer: ```suggestion
- Added `bWorkerSendHighPriority` (defaulted false) which sets the GDK worker thread priority higher than normal.
```

- Added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and ActorComponents. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
- You can now generate valid schema for classes that start with a leading digit. The generated schema class will be prefixed with `ZZ` internally.
- Handover properties will be automatically replicated when required for load balancing. `bEnableHandover` is off by default.

> - Added `bWorkerFlushAfterRPC` (defaulted false) which publishes changes to the GDK worker queue faster to allow for lower latencies. 

Reviewer: ```suggestion
- Added `bWorkerFlushAfterRPC` (defaulted false) which publishes changes to the GDK worker queue after every RPC call to allow for lower latencies. 
```

Reviewer: ?

Author: Resolved this by accident, removed the other options!

> - Added `bWorkerFlushAfterImportantEvent` (defaulted false) which publishes changes to the GDK worker queue after RPCs and property replication to allow for lower latencies. Can be used in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency.

Reviewer: ```suggestion
- Added `bWorkerFlushAfterOutgoingNetworkOp` (defaulted false) which publishes changes to the GDK worker queue after RPCs and property replication to allow for lower latencies. Can be used in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency.
```

> - Added `bWorkerFlushAfterOutgoingNetworkOp` (defaulted false) which publishes changes to the GDK worker queue after RPCs and property replication to allow for lower latencies. Can be used in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency.

Reviewer: ```suggestion
- Added `bWorkerFlushAfterOutgoingNetworkOp` (defaulted false) which publishes changes to the GDK worker queue after RPCs and property replication to allow for lower latencies. Can be used in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency at a trade-off with bandwidth.
```
